.env

```diff
@@ -0,0 +1,8 @@
+# This is an example environment file for configuring dynamic values.
+# This can be removed or changed with better solutions in the future.
+# It should *only* be kept as long as there are no sensitive values in here.
+E2E_UI_BASE_URL=http://localhost:8080/openmrs/spa/
+E2E_WS_BASE_URL=http://localhost:8080/openmrs/ws/
+E2E_USER_ADMIN_USERNAME=Admin
+E2E_USER_ADMIN_PASSWORD=Admin123
```

Comment:
Please do not put these in a file like this... They should be provided as GitHub Secrets.

Reply:
Got it!

Reply:
I've made some changes to the .env file, renaming it as example.env and removing any sensitive information. Could you please add these to Github secrets? I don't have access to the settings. Additionally, I am unsure if Github secrets are available for PRs. If they are not, we will need to commit it somewhere.
```
E2E_UI_BASE_URL=http://localhost:8080/openmrs/spa/
E2E_WS_BASE_URL=http://localhost:8080/openmrs/ws/
E2E_USER_ADMIN_USERNAME=
E2E_USER_ADMIN_PASSWORD=
E2E_LOGIN_DEFAULT_LOCATION_UUID=44c3efb0-2583-4c80-a79e-1f756a03c0a1
```

cc: @denniskigen @vasharma05